packages/models-library/src/models_library/services.py

```diff
@@ -535,108 +615,33 @@ class ServiceDockerData(ServiceKeyVersion, _BaseServiceCommonDataModel):
         ),
     )
 
+    progress_regexp: str | None = Field(
+        None,
+        alias="progress_regexp",
+        description="regexp pattern for detecting computational service's progress",
+    )
+
     class Config:
         description = "Description of a simcore node 'class' with input and output"
         extra = Extra.forbid
-        frozen = False  # it inherits from ServiceKeyVersion.
+        frozen = False  # overrides config from ServiceKeyVersion.
+        allow_population_by_field_name = True
 
         schema_extra: ClassVar[dict[str, Any]] = {
             "examples": [
-                {
-                    "name": "oSparc Python Runner",
-                    "key": "simcore/services/comp/osparc-python-runner",
-                    "type": "computational",
-                    "integration-version": "1.0.0",
-                    "progress_regexp": "^(?:\\[?PROGRESS\\]?:?)?\\s*(?P<value>[0-1]?\\.\\d+|\\d+\\s*(?P<percent_sign>%))",
-                    "version": "1.7.0",
-                    "description": "oSparc Python Runner",
-                    "contact": "smith@company.com",
-                    "authors": [
-                        {
-                            "name": "John Smith",
-                            "email": "smith@company.com",
-                            "affiliation": "Company",
-                        },
-                        {
-                            "name": "Richard Brown",
-                            "email": "brown@uni.edu",
-                            "affiliation": "University",
-                        },
-                    ],
-                    "inputs": {
-                        "input_1": {
-                            "displayOrder": 1,
-                            "label": "Input data",
-                            "description": "Any code, requirements or data file",
-                            "type": "data:*/*",
-                        }
-                    },
-                    "outputs": {
-                        "output_1": {
-                            "displayOrder": 1,
-                            "label": "Output data",
-                            "description": "All data produced by the script is zipped as output_data.zip",
-                            "type": "data:*/*",
-                            "fileToKeyMap": {"output_data.zip": "output_1"},
-                        }
-                    },
-                },
+                _EXAMPLE,
+                _EXAMPLE_W_BOOT_OPTIONS_AND_NO_DISPLAY_ORDER,
```

Comment:
I do not understand. This change precicely underlines what changed in these models because one constant is used in the other ... !

For instance
```python
_EXAMPLE_W_BOOT_OPTIONS_AND_NO_DISPLAY_ORDER = {
    **_EXAMPLE,
    "description": "this is a new description"
}    
```

See? here it is clear that the change is the description!

Reply:
Allo good, did not read it well enough.